### DISCUSSION TITLE
Re-use previous coordinator as a router

### DISCUSSION THREAD

**Discussion by Nardol** [Koenkk/zigbee2mqtt#10583] [General]
Hello,

Currently I have a ZZH as coordinator and wait for a Sonoff dongle USB+ as a replacement.
To migrate to this last one I have to set its IEEE address to the ZZH one.
But after that, if I want to re-use the ZZH as a router, if I put it within only changing the firmware there will be two devices with the same IEEE address which could cause some conflict, I suppose :slightly_smiling_face:

So what would you suggest me to do to be able to re-use it?
Use the sonoff IEEE address for the ZZH but how can I get it using cc2538-bsl?
I will use this solution as I am on Debian and I will be sure to have no inaccessible interface.

**Comment by Nardol** [comment]
Nobody has already done it?

**Comment by RenanAz** [comment]
I also need the answer to this. Im having the exact problem right now and I wish i could just change the IEEE for the ZZH. Do you know if it's possible?

**Comment by RenanAz** [comment]
I actually just made it work. I flashed the router FW with Flash Programmer 2 and also added a secondary Mac IEEE - just changed the last digit. 
This resolved the issue and allowed my old Coordinator to join the network as a router!
